### PR TITLE
Move module initialisation

### PR DESCRIPTION
**Pull request by Trildar**
Initialise PointsManager, custom gold, blink blocker handler, zone control handler, custom ability level requirements, HeroProgression, SellBlacklist, and NGP handler at start of pre-game instead of at minute 0.

### CODE REVIEW

File: game/scripts/vscripts/events.lua [hunk 1]
```diff
@@ -35,6 +35,19 @@ function GameMode:OnGameRulesStateChange(keys)
...
+    -- initialize modules
+    InitModule(PointsManager)
+    InitModule(Gold)
+    InitModule(BlinkBlock)
+    InitModule(ZoneControl)
+    InitModule(AbilityLevels)
+    InitModule(HeroProgression)
+    InitModule(SellBlackList)
+    InitModule(NGP)
+  end
```
Comment (Chronophylos) [inline game/scripts/vscripts/events.lua]: I think calling a Function in gamemode.lua is better.

Comment (Trildar) [inline game/scripts/vscripts/events.lua]: You mean put a new `OnPreGame` function in `gamemode.lua` and call it from `events.lua`?

**Comment by chrisinajar** [comment]
resolve conflicts, pretty sure it's actually your other PR so you should have no problem merging them together.

**Comment by Trildar** [comment]
Should be nicely rebased now.

**Pull request merged**